provision-osx/tasks/install_homebrew.yml

```diff
@@ -28,7 +28,6 @@
     dest: "{{ homebrew_install_path }}"
     update: no
     accept_hostkey: yes
-    depth: 1
```

Comment:
This line removal is for same issue as described here - https://github.com/ansible/ansible-modules-core/issues/5457

Git did shallow clone and couldn't checkout {{homebrew_version}} tag.